src/libRumorSpreading/RumorStateMachine.cpp

```diff
@@ -0,0 +1,180 @@
+#include "RumorStateMachine.h"
+
+#define LITERAL(s) #s
+
+namespace RRS
+{
+
+    // STATIC MEMBERS
+    std::map<RumorStateMachine::State, std::string>
```

Comment:
Did you mean to declare this as static?

Reply:
yes, its more of tied to class rather than object itself. So is declared as static.